homeassistant/components/cloud/client.py

```diff
@@ -136,8 +136,8 @@ async def get_google_config(self) -> google_config.CloudGoogleConfig:
 
         return self._google_config
 
-    async def cloud_started(self) -> None:
-        """When cloud is started."""
+    async def cloud_connected(self) -> None:
+        """When cloud is connected."""
         is_new_user = await self.prefs.async_set_username(self.cloud.username)
```

Comment:
Maybe this should happen in `cloud_started`, not in `cloud_connected`?

Reply:
What's the goal for the lifecycle of cloud with the change? There's no description here or in the library PR.

Reply:
How does the life cycle look? Is start / stop explicit actions by the client while connect / disconnect can happen due to outside reasons? Can reconnect happen while being started or does it also need a restart after reconnect?

Reply:
I can't really answer that. The point of the PR is to make the interface consistent, it does not change the life cycle.

Reply:
@ludeeus could you take a look?

Reply:
> I can't really answer that. The point of the PR is to make the interface consistent, it does not change the life cycle.

Yeah, those were just questions that came to mind after your initial question.

Reply:
That should not change. Connnected happens when we are connected to the relayer, which is required for these assistants to work.